src/testsuite/vec_int16_dummy.c

```diff
@@ -22,6 +22,59 @@
 
 #include <pveclib/vec_int16_ppc.h>
 
+vui16_t
+test_ctzh_v1 (vui16_t vra)
```

Comment:
general question: how do all of these `_v[123]` functions get called?

Reply:
This is a "compiler test", these do not "get called" or seen by the user (except as examples). 
I try things and look at what the compiler generates. In this case I saw that test_ctzh_v2 generated the best code (for GCC) and used that in the implementation of vec_ctzh.